src/containers/Chat/Chat.tsx

```diff
@@ -121,6 +121,9 @@ const Chat: React.SFC<ChatProps> = ({ contactId }) => {
     return null;
   }
 
+  // Other location.
```

Comment:
This is referenced from the last comment. The functionality from there could be moved one file up (in Chat.tsx instead of ChatMessages.tsx), but just wanted to get some feedback into whether or not this would be a better place for it.

Reply:
Yes, let's do it.